src/MiniValidation/IValidatable.cs

```diff
@@ -0,0 +1,20 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Threading.Tasks;
+
+namespace MiniValidation;
+
+/// <summary>
+/// Provides a way to add a validator for a type outside the class.
+/// </summary>
+/// <typeparam name="T"></typeparam>
+public interface IValidatable<in T>
```

Comment:
Not 100% sold on the name. Some alternatives to consider:

- `IValidate<TTarget>`
- `IValidator<TTarget>`
- `IValidatorFor<TTarget>`

Reply:
I vote for `IValidate<T>`.

Reply:
Sounds good.